.github/workflows/cpu-tests.yml

```
@@ -171,13 +171,20 @@ jobs:
           # `requirements.txt` and/or extra dependencies are missing in the Docker Conda environment
           diff env_1.yml env_2.yml
 
-      - name: Check out repository (data)
-        uses: actions/checkout@v3
-        with:
-          repository: mala-project/test-data
-          path: mala_data
-          ref: v1.7.0
-          lfs: true
+      - name: Download test data repository
+        shell: 'bash -c "docker exec -i mala-cpu bash < {0}"'
+        run: |
+          # Download test data repository from RODARE. If the version changes
+          # this URL has to be adapted (the number after /record/ and the 
+          # version have to be incremented)
+          wget "https://rodare.hzdr.de/record/2997/files/mala-project/test-data-v1.7.8.zip"
+          
+          # Once downloaded, we have to unzip the file. The name of the root
+          # folder in the zip file has to be updated for data repository
+          # updates as well - the string at the end is the hash of the data 
+          # repository commit. 
+          unzip test-data-v1.7.8.zip
```

Add `-q` flag for quiet output to avoid cluttering the log files with noise:

```suggestion
          unzip -q test-data-v1.7.8.zip
```

---

Ah good point, let's do that!